src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+    {
+      person.isMarried ? (
+        <p className="Person__partner">
+          {person.partnerName}
+          {' is my '}
+          {person.sex === 'm' ? 'wife' : 'husband'}
+        </p>
+      ) : (
+        <p className="Person__partner">I am not married</p>
+      )
+    }
```

Comment:
Try not to use nested ternary oparator, you can move this part person.sex === 'm' ? 'wife' : 'husband' to separate variable for example pertner and use this variable here.
Also you don't need to create two `<p>` tags, you can create one and change only content using ternary

Reply:
thanks)